harry-integration/test/harry/model/HistoryBuilderIntegrationTest.java

```diff
@@ -1,192 +0,0 @@
-/*
- *  Licensed to the Apache Software Foundation (ASF) under one
- *  or more contributor license agreements.  See the NOTICE file
- *  distributed with this work for additional information
- *  regarding copyright ownership.  The ASF licenses this file
- *  to you under the Apache License, Version 2.0 (the
- *  "License"); you may not use this file except in compliance
- *  with the License.  You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-
-package harry.model;
-
-import java.util.HashSet;
-import java.util.Random;
-import java.util.Set;
-import java.util.function.Supplier;
-
-import org.junit.Assert;
-import org.junit.Test;
-
-import harry.core.Configuration;
-import harry.core.Run;
-import harry.ddl.SchemaGenerators;
-import harry.ddl.SchemaSpec;
-import harry.dsl.HistoryBuilder;
-import harry.operations.CompiledStatement;
-import harry.operations.Query;
-import harry.operations.QueryGenerator;
-import harry.reconciler.Reconciler;
-import harry.util.TestRunner;
-import harry.visitors.MutatingVisitor;
-import harry.visitors.MutatingRowVisitor;
-import harry.visitors.ReplayingVisitor;
-
-public class HistoryBuilderIntegrationTest extends ModelTestBase
```

Comment:
nit: Did we mean to delete this file?

Reply:
Yes, this is for the old, bad history builder!